bycycle/burst/cycle.py

```diff
@@ -62,6 +62,18 @@ def detect_bursts_cycles(df_features, amp_fraction_threshold=0., amp_consistency
     -----
     * The first and last period cannot be considered oscillating if the consistency measures are
       used.
+
+    Examples
+    --------
+    Apply thresholding for consistency burst detection.
+
+    >>> from bycycle.features import compute_burst_features, compute_shape_features
+    >>> from neurodsp.sim import sim_bursty_oscillation
+    >>> fs, f_range = 500, (8, 12)
+    >>> sig = sim_bursty_oscillation(10, fs, 10)
```

Comment:
for clarity on what the inputs are, I think I would write out `freq=10` in the inputs to sim_bursty_oscillation. 
also, since `f_range` is only used once, it could be written in the inputs to the func below (as `f_range=(8, 12)`), then we don't need double assignment on a line.